### PR TITLE
react router

### PR DESCRIPTION
**Pull request by VelzckC0D3**
# Math Magicians -  `React` `ES6` `API`

## ✨ _Requested Implementations_ ✨
✔️ _Created a website that have 3 pages: `Home`, `Calculator` & `Quote`_
✔️ _Followed general layout_
✔️ _Added my own style to improve the look and feel_
✔️ _Every page has its own `Route`_
✔️ _**The navigation links to all 3 `Routes`** & they are displayed correctly_
✔️ _Used React Router to make the routes work_

#  💡 _Thank you for your review!_ 💡

### CODE REVIEW

**Comment by VelzckC0D3** [comment]
✨ Please note that the linters that are failed are the ones from Netlify, but the linters from the Workflow are working correctly, also, it says that it's failed but the Netlifty deploy is ok ✨
[Netlify Deploy](https://velzck-magician.netlify.app/)

File: src/App.js [hunk 1]
```diff
@@ -1,15 +1,29 @@
...
-  </>
-);
+function App() {
+  return (
+    <Router>
+      <>
+        <Navbar />
+        <div className="App">
+          <h1>Velzck&#39;s Math Magicians</h1>
+        </div>
```
Comment (Tufahel) [inline src/App.js]: image
- As per the project requirements `the general layout should match with the wireframe`. In the wirefram wireframe the `title` should be aligned with the navigation `menu`.
image
- But I can see you have added this below the navigation `menu`.
- I recommend you use it inside the `Navbar.js` and align it with the menu as per the given wireframe.
- [ ] Kindly fix this issue.

**Review by Tufahel — CHANGES_REQUESTED** [review]
# Status: Required Changes :recycle:
Hi @VelzckC0D3 :wave:  

Good job so far! 👍 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights 🌟

Alt Text

✔️ Professional pull request
✔️ Correct Git-flow
✔️ Linter checks are passing
✔️ App is working fine
✔️ Awesome UI design.

## Required Changes ♻️

***Please kindly check the comments below.***

## Optional suggestions 🤷‍♂️

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

## Important Notes 📝
- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js (hunk 1 above)
Comment (VelzckC0D3) [reply src/App.js]: Done!, thank you @Tufahel

**Review by Tufahel — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:

Hi @VelzckC0D3 :wave: 
 
Well done on implementing all of the changes! 👍  Your project is complete now!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 👏🏻



:heavy_check_mark: Linter checks are passing.
:heavy_check_mark: Descriptive pull request.
:heavy_check_mark: UI looks great.
:heavy_check_mark: Correct Git-flow.

## Optional suggestions 🤷‍♂️

- [ ] Kindly use the capital letter as the first letter of the commit message as it is professional and looks smart.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

## Important Notes 📝
- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.
- You can mail me at [Mail](mailto:tufahel96@gmail.com) or ask any question at Slack, `Name: Tufahel`

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**